### PR TITLE
Train and Test Pipeline

### PR DESCRIPTION
**Pull request by jackzhy96**

### CODE REVIEW

**Comment by jabarragann** [comment]
Hi Jack,

Thanks for the pull request I am reviewing it right now. Currently, your pull request is targeting my main branch. Is there a way of changing it to the devel branch?

File: scripts/README.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+# How to Run the Scripts for Training and Testing
+Contributor: Juan Antoni Barragan & Haoying(Jack) Zhou
```
Comment (jabarragann) [inline scripts/README.md]: Small typo on my name! Missing an o.

Also, I think we should move your readme file to the docs folder and rename it to `running_train_and_test_scripts.md` or something similar.

File: scripts/README.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+│   ├── segmented
+│   │   ├── xxx.png
+│   │   ├── ...
+├── dataset_config.yaml
+```
+
+## Create Binary Segment
+
+```bash
+python3 create_binary_segment.py -i <src_foler>
```
Comment (jabarragann) [inline scripts/README.md]: Typo in here!

File: scripts/copy_rename_files.py [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+    # data_path = os.path.join(dynamic_path, 'monai_data', 'AMBF_DATASETS')
+    # out_folder = os.path.join(dynamic_path, 'monai_data', 'AMBF_DATASETS_New')
+
+    file_reorganizer = FilesRearrange(data_path, out_folder, flag_split)
+
+    file_reorganizer.main()
+
+    # Copy the yaml file
+    yaml_name = 'dataset_config.yaml'
+    src_yaml_file = os.path.join(dynamic_path, 'monai_data', yaml_name)
```
Comment (jabarragann) [inline scripts/copy_rename_files.py:143]: This seems like something very specific to your setup. 

I would say the yaml file you be kept in the root directory of the dataset and then copied from there to the new folder.

File: scripts/create_binary_segment.py [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+class BinarySegment:
+    def __init__(self, src_folder: str):
+        '''
+        :param src_folders: source folders path
+        '''
+        self.src_folder = src_folder
+        self.subfolder_name = 'segmented'
+        self.outfolder_name= 'binary_segmented'
+
+        assert os.path.exists(os.path.join(self.src_folder, self.subfolder_name)), 'Segmented Image folder does not exist!'
```
Comment (jabarragann) [inline scripts/create_binary_segment.py]: Let's improve this error message to something like:

`Segmented Image folder does not exist! Does your input path include the split that you want to process? For example ./AMBF_DATASETS_NEW/train`

**Review by jabarragann — CHANGES_REQUESTED** [review]

**Comment by jackzhy96** [comment]
The previous problems have been addressed, PR change to devel branch

**Pull request merged**